app/build.gradle

```diff
@@ -27,7 +27,13 @@ android {
         versionName "1.0.3alpha"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
         vectorDrawables.useSupportLibrary = true
-        manifestPlaceholders = [appName: app_name]
+        each { type ->
+            type.buildConfigField "String", "GOOGLE_PLACES_API_KEY", "\"" + GOOGLE_PLACES_API_KEY + "\""
```

Comment:
Use envoronment variables please, or else we won't have any way of injecting the API key at build time

Reply:
sorry, I have corrected